test/common/tests.go

```diff
@@ -48,6 +48,16 @@ var (
 				{"A15 - Verify Stateful Set resources have the expected replicas", TestStatefulSetsExpectedReplicas},
 				{"A26 - Verify Sendgrid Credentials Are Configured Properly", TestSendgridCredentialsAreValid},
 				{"C01 - Verify Alerts are not pending or firing apart from DeadMansSwitch", TestIntegreatlyAlertsPendingOrFiring},
+<<<<<<< HEAD
+=======
+				{"C04 - Verify Alerts exist", TestIntegreatlyAlertsExist},
+				{"C10B - Verify Prometheus blackbox targets", TestAdditionalBlackboxTargets},
+				{"C08B - Verify alert links to SOPs", TestSOPUrls},
+				{"E01 - Verify Middleware Grafana Route is accessible", TestGrafanaExternalRouteAccessible},
+				{"E02 - Verify that all dashboards are installed and all the graphs are filled with data", TestDashboardsData},
+				{"E03 - Verify middleware dashboards exist", TestIntegreatlyMiddelewareDashboardsExist},
+				{"E05 - Verify Grafana Route returns dashboards", TestGrafanaExternalRouteDashboardExist},
+>>>>>>> e06ff4928 (MGDAPI-3461-c14b-test-automation-adressing-PR-comments)
```

Comment:
same here, this is not a valid go code, these are git marks for conflict resolution